Arelle/Arelle
# Rename ESEF 2024 disclosure system to draft

#### Reason for change
resolves #1413 

#### Description of change
* Rename ESEF 2024 disclosure system to specify that it is in draft status (until conformance suite is released.)
* Disable `ESEF.2.6.1.reportIncorrectlyPlacedInPackag` validation for 2024 that was removed in the 2024 filing manual.
* Fix xbrli type check.

#### Steps to Test
* Use the doc attached to the above issue.
* `python arelleCmdLine.py --file TC3_invalid.xbri --plugins validate/ESEF --disclosureSystem esef-2024-draft --validate`
* Verify that there are no errors reported for xbri report packages not being permitted.

**review**:
@Arelle/arelle
